src/backend-api/Controllers/EmployeesController.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
```

Comment:
Wasn't initially expecting to see a controller in this commit - I assume this was largely scaffolded, yeah?

Reply:
The ValuesController was part of the scaffold. I made the EmployeesController because the subtask said we wanted to get Active Directory working and also test out and make sure Entity Framework correctly made the models.

Reply:
Right on - since there may have been some potential overlap between this subtask and https://jira.cqlcorp.com/browse/IIWA-26 I might recommend taking a peek through 26 and updating any subtasks that may no longer be necessary. I would leave the acceptance criteria as-is though.